docs/internals/ARCHITECTURE.md

```diff
@@ -405,7 +405,7 @@ time to verify O(1) operation).
 
 ### Synchronous Execution
 
-StateMachine's `commit` function, the one that actually implements double-entry accounting, is
```

Comment:
Emphasis is intentional: the _whole_ tiegrbeetle implements double-entry accounting, but commit is where the business logic proper is encoded.